### DISCUSSION TITLE
Is it possible to select columns with a matching name (.e.g to get all bid columns)

### DISCUSSION THREAD

**Discussion by andrew-mcnaught** [grafana/grafana-csv-datasource#66] [Q&A] [answered]
Hey, thanks for you work on the csv loader, it looks very good so far.

I was wondering if there is a way of selecting all columns with a similar naming convention. For example the following works
![image](https://user-images.githubusercontent.com/84084967/117989214-09432a00-b334-11eb-9d89-277dba2afdb0.png)
But is it possible to query in some way (/Bid/ or something) to get all of the columns with Bid* as such?

Thanks,
Andrew

**Comment by marcusolsson** [comment, marked as answer]
Currently no, but you're not the first one to ask. I hinted that we could add regex support in the field name here: https://github.com/marcusolsson/grafana-csv-datasource/discussions/63

Could you add a feature request for this maybe?

**Reply by andrew-mcnaught** [reply]
Just added, thanks very much for the quick response!
